### PR TITLE
Add vin column to OperationResult table

### PR DESCRIPTION
**Pull request by calumats**

### CODE REVIEW

File: core/src/main/resources/db/migration/V3__add_operationresult_vin.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ALTER TABLE OperationResult
+ADD vin varchar(64) NOT NULL,
+DROP PRIMARY KEY,
+ADD CONSTRAINT PK_operationresult PRIMARY KEY (vin, id, update_request_id);
```
Comment (jerrytrieu) [inline core/src/main/resources/db/migration/V3__add_operationresult_vin.sql]: Let's just change V2 sql

**Comment by calumats** [comment]
Added a commit to fix some broken unit tests. Not sure if that was broken by my last PR introducing the Firmware table, but it's fixed now.

**Comment by jerrytrieu** [comment]
Oh, my last commit on the genivi branch broke the unit tests. Let's separate your commit into that one.

**Comment by jerrytrieu** [comment]
UI needs update, show operation-results on the vehicle page.

**Comment by calumats** [comment]
Done

**Pull request merged**